openquake/hazardlib/gsim/cauzzi_2014scaled.py

```diff
@@ -0,0 +1,74 @@
+# -*- coding: utf-8 -*-
+# vim: tabstop=4 shiftwidth=4 softtabstop=4
+#
+# Copyright (C) 2020 GEM Foundation
+#
+# OpenQuake is free software: you can redistribute it and/or modify it
+# under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# OpenQuake is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with OpenQuake. If not, see <http://www.gnu.org/licenses/>.
+
+"""
+Module exports :class:`CauzziEtAl2014Eurocode8scaled`,
+"""
+import numpy as np
+# standard acceleration of gravity in m/s**2
+from openquake.hazardlib.gsim.base import CoeffsTable
+from openquake.hazardlib.gsim.cauzzi_2014 import CauzziEtAl2014
+
+
+class CauzziEtAl2014Eurocode8scaled(CauzziEtAl2014):
+    """
+    Implements GMPE developed by Carlo Cauzzi et al (2014) and published
+    as C.Cauzzi, E. Faccioli, M. Vanini and A. Bianchini (2014) "Updated
+    predictive equations for broadband (0.0 - 10.0 s) horizontal response
+    spectra and peak ground motions, based on a global dataset of digital
+    acceleration records", Bulletin of Earthquake Engineering, In Press
+
+    Spectral acceleration (SA) values are obtained from displacement response
+    spectrum  (DSR) values (as provided by the original equations) using the
+    following formula ::
+
+        SA = DSR * (2 * π / T) ** 2
+    """
+    def _get_site_amplification_term(self, C, vs30):
```

Comment:
This method seems to me identical to https://github.com/gem/oq-engine/blob/master/openquake/hazardlib/gsim/cauzzi_2014.py#L471

Reply:
Removed